### ISSUE TITLE
Implement simple user's paths list

### ISSUE THREAD

**Issue by karolsojko** [x-team/unleash#224]
When you go from profiles list to a specific user you should see a simple list of his paths and goals in them.

here's the API Doc for retrieving the paths for a user: http://paths.unleash.x-team.com/api/doc#get--api-v1-paths.{_format}

so you should make a call like this: `http://paths.unleash.x-team.com/api/v1/paths?userId={id}`

The page should display a simple list (no styling) of paths and goals in them - something similar to the skills page but sort of multiple lists.

Note: styling of this page will be done in a separate issue.

**Comment by kdemoya** [comment]
Taking over this one.

**Comment by karolsojko** [comment]
just a quick update - @kdemoya please review https://github.com/x-team/unleash/blob/develop/CONTRIBUTING.md before starting ;)
